### PR TITLE
Improve the description of pitch_adjust

### PR DESCRIPTION
**Pull request by daschuer**
This also fixed the pre-commit issues. 
The pending Sphinx failure is fixed here: https://github.com/mixxxdj/manual/pull/481

### CODE REVIEW

**Comment by ronso0** [comment]
Let's please apply the pre-commit fixes in a separate PR.

**Comment by daschuer** [comment]
now the black update lives here: https://github.com/mixxxdj/manual/pull/482

File: source/chapters/appendix/mixxx_controls.rst [hunk 1]
```diff
@@ -1818,11 +1818,11 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
                    [PreviewDeckN],pitch_adjust
                    [SamplerN],pitch_adjust
 
-   Adjust the pitch in addition to the :term:`tempo` slider pitch.
+   Adjusts the pitch in addition to the :term:`tempo` slider pitch and keylock. It is reset after loading a new track.
 
    This is a :ref:`ControlPotMeter control <appendix-mixxxcontrols-controlpotmeter>`.
 
-   :range: -3.0..3.0
```
Comment (ronso0) [inline source/chapters/appendix/mixxx_controls.rst:1825]: 7882443 adds this suffix also to the `pitch` range

Comment (daschuer) [reply source/chapters/appendix/mixxx_controls.rst:1825]: Merged, thank you.

**Comment by ronso0** [comment]
Thank you!

**Pull request merged**